### ISSUE TITLE
Block intermod support of specified key materials.

### ISSUE THREAD

**Issue by koiNoCirculation** [GTNewHorizons/GT-New-Horizons-Modpack#6093]
Materials like:
Titanium, Platinum, Tungsten, Iridium, Osmium, etc
are key materials of:
EV, EV, IV, IV, LUV/ZPM
which can be bypassed through adding some mods, like RotaryCraft and Mechanism.
Through bypassing approach, these materials can be processed without EBF or LCR cycling recipes.
The bypassing can be blocked by delibrately adding typos to oreDict of  related ores and materials.

**Comment by GTNH-Afx237v7** [comment]
Rotarycraft and Mekanism are not part of the pack. Obviously, certain mechanics can be circumvented by adding/removing or otherwise altering the pack content. However I don’t see how this should be handled, as that would require precautions against any possible mod and most likely cause more issues than keeping the pack as it is. If someone wants to add other mods to their instance, that would be their choice then.

**Comment by koiNoCirculation** [comment]
@GTNH-Afx237v7 
Emm, you know, dustTitanium, orePlatinum, oreScheelite, ingotTitanium, ingotPlatinum etc, are naturally used by english-speaking modders, so it's possible for other mods, which isn't part of pack to accidentally add some easy recipes to your pack. 
They added RotaryCraft for bypassing rare metal processing changes added by bartworks since 2080, because they thought the nerf of smelting recipe of platinum metal dust was shitty. 
The bypass of Titanium, Tungsten and Iridium processing are just accidentally discovered when adding RotaryCraft to the pack.

**Comment by repo-alt** [comment]
I seriously don't see how is that a GTNH problem.

**Comment by draknyte1** [comment]
> The bypassing can be blocked by delibrately adding typos to oreDict of related ores and materials.

Literally the dumbest suggestion ever. 

We don’t support non-standard pack configurations and if problems arise from adding/removing mods, it’s definitely not on us to resolve them. 😂🤦🏻‍♀️
